PROTOCOL.md

```diff
@@ -39,6 +32,10 @@ This documentation describes the messages in each layer.
     - [ResendResponseResent](#resendresponseresent)
     - [ResendResponseNoResend](#resendresponsenoresend)
     - [ErrorResponse](#errorresponse)
+    - [StatusMessage](#statusmessage)
```

Comment:
We should somehow more clearly separate the messages used in node-to-node protocol and node-to-tracker protocol, now they're just part of the same bunch of message types.

Actually, maybe the "Layer" word is not the best choice for the message categories. We could have, for example:
- Control Messages
- Stream Messages
- Tracker Messages

Reply:
We could just arrange the messages into those categories in this document?

Reply:
I agree that there is no requirement for the messages to be in the same numbering space, however I don't see a requirement for them not to be either. For practical purposes (such as avoid confusion while debugging etc.) it can just make life easier to have separate number for each message type.

Reply:
> We could just arrange the messages into those categories in this document?

Yeah sounds good to me!

> For practical purposes (such as avoid confusion while debugging etc.) it can just make life easier to have separate number for each message type.

Yeah true, one point though, someone creating a new message type shouldn't need to check all the other categories to see whether the same number has already been taken in another category. I don't have a problem if we try it